### PR TITLE
Add isEmpty macro

### PR DESCRIPTION
**Pull request by ctjhoa**
This PR add an global `isEmpty` which is a wrapper for `Ember.isEmpty`.
I'm not sure to doing it the right way and also about the doc in the readme.

### CODE REVIEW

File: tests/integration/is-empty-test.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      source: { size: 1 }
+    },
+    strictEqual: false
+  });
+});
+
+test('not empty with object size function 0', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
```
Comment (kellyselden) [inline tests/integration/is-empty-test.js]: Can you add a test of composing? `isEmpty([])` and `isEmpty(collect(1, 2))`

File: tests/integration/is-empty-test.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    properties: {
+      source: [0, 1, 2]
+    },
+    strictEqual: false
+  });
+});
+
+test('not empty with eol & tab', function(assert) {
```
Comment (kellyselden) [inline tests/integration/is-empty-test.js]: Most of these tests can be removed. We don't need to test the implementation of Ember.isEmpty, just that our macro works in ways we expect.

File: README.md [hunk 3]
```diff
@@ -109,6 +109,7 @@ import { nameOfMacro } from 'ember-awesome-macros';
 ##### Object
 * [`getBy`](#getby)
 * [`hash`](#hash)
+* [`isEmpty`](#isEmpty)
```
Comment (kellyselden) [inline README.md]: can you lowercase the link?

File: README.md [hunk 4]
```diff
@@ -666,6 +667,29 @@ value2: hash('source1', 'source2'), // { source1: 'my value 1', source2: 'my val
...
+sourceObject2: { size: 0 },
+
+valueString1: isEmpty('sourceString1'), // true
+valueString2: isEmpty('sourceString2'), // false
+
+valueArray1: isEmpty('sourceArray1'), // true
+valueArray2: isEmpty('sourceArray2'), // false
+
+valueObject1: isEmpty('sourceObject1'), // false
+valueObject2: isEmpty('sourceObject2'), // true
```
Comment (kellyselden) [inline README.md]: can you add some examples of composing and raw values?

**Comment by ctjhoa** [comment]
@kellyselden I have fixed your comments tell me if it's better for you.

**Comment by ctjhoa** [comment]
@kellyselden up

File: README.md (hunk 4 above)
Comment (mriska) [reply README.md]: @kellyselden would raw values ever be interesting with `isEmpty`? That seems like a feature that would never be used.

Comment (kellyselden) [reply README.md]: You are right. Raw values would be a pretty useless example.

File: README.md [hunk 5]
```diff
@@ -711,6 +712,33 @@ value2: hash('source1', 'source2'), // { source1: 'my value 1', source2: 'my val
 value3: hash('source1', { prop2: 'source2' }) // { source1: 'my value 1', prop2: 'my value 2' }
 ```
 
+##### `isEmpty`
+wraps [`Ember.isEmpty`](https://emberjs.com/api/ember/3.0/functions/@ember%2Futils/isEmpty)
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply README.md]: done

**Comment by kellyselden** [comment]
Looks great. Thanks again.

**Pull request merged**